benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/benchmarks/animation/AnimatedVisibility.kt

```diff
@@ -38,7 +38,7 @@ fun AnimatedVisibility() {
             AnimatedVisibility(showImage) {
                 transition = this.transition
                 Image(
-                    painterResource(Res.drawable.compose_multiplatform),
+                    painterResource(Res.drawable.img),
```

Comment:
I guess compose_multiplatform.xml is not used anymore and can be removed from the resources (and thus we can name png as compose_multiplatform.png).

Reply:
fixed